src/js/stores/transforms/AppFormModelToFieldTransforms.js

```diff
@@ -1,15 +1,40 @@
 const AppFormModelToFieldTransforms = {
-  constraints: (constraints) => constraints
-    .map((constraint) => constraint.join(":"))
-    .join(", "),
-  env: (rows) => Object.keys(rows)
-    .map((rowKey, i) => {
-      return {
-        key: rowKey,
-        value: rows[rowKey],
-        consecutiveKey: i
-      };
-    }),
+  constraints: (constraints) => {
+    return constraints
+      .map((constraint) => constraint.join(":"))
+      .join(", ");
+  },
+  dockerPortMappings: (mappings) => {
+    return mappings
+      .map((row, i) => {
+        row.consecutiveKey = i;
+        return row;
+      });
+  },
+  dockerParameters: (parameters) => {
+    return parameters
+      .map((row, i) => {
+        row.consecutiveKey = i;
+        return row;
+      });
+  },
+  containerVolumes: (volumes) => {
+    return volumes
+      .map((row, i) => {
+        row.consecutiveKey = i;
+        return row;
+      });
+  },
+  env: (rows) => {
+    return Object.keys(rows)
```

Comment:
I wanted to lazy.js this, but this unexpected bahviour prevents me from this:
https://github.com/dtao/lazy.js/issues/164